src/preprocessor.ts

```diff
@@ -58,7 +60,15 @@ export function process(
 
   const modified = injectSourcemapHook(outputText);
 
-  cacheFile(jestConfig, filePath, modified);
+  if (tsJestConfig.enableInternalCache === true) {
+    // This config is undocumented.
+    // This has been made configurable for now to ensure that
+    // if this breaks something for existing users, there's a quick fix
+    // in place.
+    // If this doesn't cause a problem, this if block will be removed
+    // in a future version
+    cacheFile(jestConfig, filePath, modified);
```

Comment:
I'd like a date on this comment so "in a future version" doesn't just lie around forever.

Reply:
I put it that way because I wasn't sure what a good time-frame would be. Any suggestions?